### DISCUSSION TITLE
Can someone build this for SQL?

### DISCUSSION THREAD

**Discussion by 27medkamal** [0no-co/gql.tada#105] [Ideas]
Great project and it's exactly what the graphql ecosystem needed.

However, I think the appeal of an SQL equivalent is far greater. If a similar engine existed for SQL, I and many others like myself wouldn't use an ORM.

Just an idea!

**Comment by kitten** [comment]
While this isn't the exact same, I believe what you're looking for is likely Kysely: https://www.kysely.dev/
This isn't like Drizzle, which is an ORM, but instead a typed SQL builder.
While it's not the same as `gql.tada`’s approach, it's still similar enough in spirit

**Reply by 27medkamal** [reply]
Kysely is probably the next best thing you're right, but having the ability to write sql queries that can also run directly against the database would be great for a variety of reasons. Anyway, I understand that it's not what this project is about, but thought I'd throw the idea out there in case someone was interested in building it.
